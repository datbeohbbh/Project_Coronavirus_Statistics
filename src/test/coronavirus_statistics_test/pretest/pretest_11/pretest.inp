29 30
q 1 13 433
? 6
q 1 7 290
? 4 2
q 3 7 49
? 4 1
q 1 16 285
? 6
? 6
q 2 13 399
q 3 16 167
q 2 16 49
? 6
? 7
q 3 7 204
q 1 23 11
q 2 16 14
? 4 3
q 1 14 162
q 2 7 30
q 1 17 358
? 4 5
q 2 7 2
? 6
? 4 3
? 6
? 4 1
? 4 3
? 4 6
? 4 4