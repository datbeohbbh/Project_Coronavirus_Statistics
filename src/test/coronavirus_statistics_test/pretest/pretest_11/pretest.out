Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 433
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 241
-> Has recovered : 0
-> Was dead : 49

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 399 people

Query ? 7 :
The city with the highest death is city with the index 16
Quantity: 167 people

Query ? 4 3 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 37
-> Has recovered : 0
-> Was dead : 253

Query ? 4 5 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 162
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 399 people

Query ? 4 3 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 34
-> Has recovered : 399
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 399 people

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 5
-> Has recovered : 32
-> Was dead : 253

Query ? 4 3 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 34
-> Has recovered : 399
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 358
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 55
-> Has recovered : 63
-> Was dead : 167

